cal-heatmap.js

```diff
@@ -2503,18 +2593,28 @@ CalHeatMap.prototype = {
 				_callback({});
 				return true;
 			} else {
+				var url = this.parseURI(source, startDate, endDate);
```

Comment:
I think these changes are unrelated to the tooltip feature, and deserve a separate pull request